requirements/main.txt

```diff
@@ -1475,6 +1488,10 @@ text-unidecode==1.3 \
     --hash=sha256:1311f10e8b895935241623731c2ba64f4c455287888b18189350b67134a822e8 \
     --hash=sha256:bad6603bb14d279193107714b288be206cac565dfa49aa5b105294dd5c4aab93
     # via python-slugify
+tqdm==4.65.0 \
+    --hash=sha256:1871fb68a86b8fb3b59ca4cdd3dcccbc7e6d613eeed31f4c332531977b89beb5 \
+    --hash=sha256:c4f53a17fe37e132815abceec022631be8ffe1b9381c2e6e30aa70edc99e9671
+    # via b2sdk
```

Comment:
I think this dependency is a bug in b2sdk, https://github.com/Backblaze/b2-sdk-python/issues/387

Reply:
Thanks for taking a look @ppolewicz, if b2sdk drops it in the future, we'll pick it up when we pull that release.